prow/config/tide.go

```diff
@@ -97,6 +97,11 @@ type Tide struct {
 	// Leave this blank to disable this feature and save 1 API token per sync loop.
 	BlockerLabel string `json:"blocker_label,omitempty"`
 
+	// SquashLabel is an optional label that is used to identify PRs that should
+	// always be squash merged.
+	// Leave this blank to disable this feature and save 1 API token per sync loop.
```

Comment:
isn't it actually 1 token per PR per sync loop?

Reply:
er actually one token per merged PR?

Reply:
Yes, you're right. One extra token per merged PR. Will fix comment.